chsdi/models/s3_client.py

```diff
@@ -27,8 +30,11 @@ def get_file_from_bucket(bucket_name, file_name):
     try:
         response = conn.get_object(Bucket=bucket_name,
                                    Key=file_name)
+    except boto_exc.NoCredentialsError as c:
```

Comment:
Would it probably also help to do something like a `log.error()` or `log.debug()` or the like with the anonymized details on the bucket and to not have any details on the bucket in the raised error message itself?

Reply:
Same, the error is catched after in https://github.com/geoadmin/mf-chsdi3/pull/3898/files#diff-ef400a87c6b46b2854e0f728421924a1111c8eedbc04056c6b3ae79bfc531e21R447